airflow/contrib/sensors/bigquery_sensor.py

```diff
@@ -50,7 +50,7 @@ def __init__(self,
                  project_id,
                  dataset_id,
                  table_id,
-                 bigquery_conn_id='bigquery_default_conn',
+                 bigquery_conn_id='bigquery_default',
```

Comment:
Other operators for GCP use `google_cloud_default` as a default connection id.  Maybe we should go in this direction?

Reply:
Agree with @mik-laj :-)

Reply:
I started working in this direction, but I did not have time to finish it. If you want, you can base your work on this change.
https://github.com/PolideaInternal/airflow/pull/42/files

Reply:
@mik-laj @Fokko Agreed.

@mik-laj Nice initiative for your changes. Will you have time to continue working on it? If not, I would love to take the whole task. Cheers.

Reply:
@ryanyuan I am currently dealing with other changes. Feel free and take the code / task.

Reply:
@mik-laj Cool! :thumbsup: